include/swift/Basic/Features.def

```diff
@@ -98,6 +98,7 @@ SUPPRESSIBLE_LANGUAGE_FEATURE(
     FreestandingExpressionMacros, 382, "Expression macros",
     true)
 LANGUAGE_FEATURE(AttachedMacros, 389, "Attached macros", true)
+LANGUAGE_FEATURE(MoveOnly, 390, "noncopyable types", true)
```

Comment:
that's incredibly subtle 🤔 

Does the freestanding in the next like also mean it is wrong then?

Reply:
the secret sauce is here: https://github.com/apple/swift/blob/294342a1d3517ce4477266ab275df4de8a97c3e1/lib/Parse/ParseIfConfig.cpp#L553

that string switch determines whether `#if $MoveOnly` evaluates to true.